File: trial_division.swift
// http://rosettacode.org/wiki/Primality_by_trial_division

func isPrime(n:Int) -> Bool {
  for var i = 3; i < n; i = i + 2 {
    if n % i == 0 {
      return false
    }
  }
  return true
}

println(isPrime(37))
println(isPrime(38))
println(isPrime(39))

File: trial_division.py
#!/usr/bin/env python
# http://rosettacode.org/wiki/Primality_by_trial_division

def is_prime(n):
  for i in xrange(3,n,2):
    if n % i == 0:
      return False
  return True

print(is_prime(37))
print(is_prime(38))
print(is_prime(39))
